hera_pspec/utils.py

```diff
@@ -1451,7 +1452,7 @@ def uvp_noise_error(uvp, auto_Tsys=None, err_type='P_N', precomp_P_N=None, P_SN_
                         Tflag = np.all(Tflag, axis=-1)
                         # interpolate to appropriate LST grid
                         if np.count_nonzero(~Tflag) > 1:
-                            Tsys = interp1d(lsts[~Tflag], Tsys[~Tflag], kind='nearest', bounds_error=False, fill_value='extrapolate')(lst_avg)
+                            Tsys = interp1d(auto_lst_avg[~Tflag], Tsys[~Tflag], kind='nearest', bounds_error=False, fill_value='extrapolate')(lst_avg)
```

Comment:
Is it better to interpolate on tinds1[~Tflag] and tinds2[~flag] separately and then multiply?

Reply:
That seems smart. I can do that.

Reply:
Ok, done.